README.md

```diff
@@ -10,4 +10,53 @@ Quantify Treasury Risk exposure with Whip's Risk Analytics Dashbord. Identify ri
 
 - Real-time Treasury Risk Analysis
 - Access to Risk Mitigation Strategies
-- Backtest Strategies before committing
\ No newline at end of file
+- Backtest Strategies before committing
+
+## Local installation
+
+First:
+
+```sh
+cp backend/app.env.template backend/app.env
+```
+
+then open `backend/app.env` and edit the environment variables with the right API keys and such.
+
+
+For development work, you probably want to run the frontend dev server in your own shell:
+
+```sh
+# Shell 1:
+docker compose --profile backend up
+# Shell 2:
+cd frontend && npm run dev
```

Comment:
maybe we should mention that the user should install frontend deps with `npm i` before running npm scripts